### PR TITLE
Added a PR template

### PR DESCRIPTION
**Pull request by cyapa**
I added a PR template so that we can improve the readability of the PRs and improve the productivity in reviewing the PRs :)
Appreciate your thoughts on this 👍

### CODE REVIEW

**Review by elguitar — APPROVED** [review]
Seems like a good starting point for a PR template. I don't see any reason to not merge this :)

I would suggest squashing the commits into one before merging. With GH you can use the `squash and merge` button. If you want to train you inner git-ninja, you can run git reset to main while preserving index (`git reset --soft main`, and then committing with a new message `git commit -m "Message here"`)

**Comment by cyapa** [comment]
> Seems like a good starting point for a PR template. I don't see any reason to not merge this :)
> 
> I would suggest squashing the commits into one before merging. With GH you can use the `squash and merge` button. If you want to train you inner git-ninja, you can run git reset to main while preserving index (`git reset --soft main`, and then committing with a new message `git commit -m "Message here"`)

Yep, Im a big fan of `squash and merge` as well 💯

**Pull request merged**
